manifests/viewer.pp

```diff
@@ -1,3 +1,5 @@
+# A class to configure the diff viewer webservice. For full details, see
```

Comment:
pelease prefix this with `# @summary` so puppet-strings can pick the information up and place it into a REFERENCE.md